api/embedded/token.go

```diff
@@ -47,3 +55,95 @@ func (t *TokenApi) GetByZts(zts types.ZenonTokenStandard) (*api.Token, error) {
 	}
 	return &result, nil
 }
+
+// Contract methods
+func (t *TokenApi) IssueToken(
+	tokenName string,
+	tokenSymbol string,
+	tokenDomain string,
+	totalSupply *big.Int,
+	maxSupply *big.Int,
+	decimals uint8,
+	mintable bool,
+	burnable bool,
+	utility bool) (*nom.AccountBlock, error) {
```

Comment:
```suggestion
	tokenName,
	tokenSymbol,
	tokenDomain string,
	totalSupply,
	maxSupply *big.Int,
	decimals uint8,
	mintable,
	burnable,
	utility bool) (*nom.AccountBlock, error) {
```

You can shorten method definitions like this.

Reply:
thanks!

Reply:
for me, it's a tossup for what is more readable